CHANGELOG.md

```diff
@@ -1,20 +1,22 @@
-x.y.z Release notes (yyyy-MM-dd)
+3.18.1 Release notes (yyyy-MM-dd)
```

Comment:
This is set as part of making a release, and changing it before that breaks the script that automatically sets it. 3.18.1 is also not the correct version number for the next release.